tools/bug-report/pkg/kubectlcmd/kubectlcmd.go

```diff
@@ -34,31 +34,50 @@ import (
 
 const (
 	// maxRequestsPerSecond is the max rate of requests to the API server.
-	maxRequestsPerSecond = 10
-	// maxLogFetchConcurrency is the max number of logs to fetch simultaneously.
-	maxLogFetchConcurrency = 10
```

Comment:
I didn't see a lot of value of keeping both these constants as they're the same, and didn't want to add 2 flags for UX reasons, using the rps limit for log fetch concurrency seems like a reasonable choice